core/src/main/scala/com/microsoft/azure/synapse/ml/causal/DoubleMLParams.scala

```diff
@@ -3,38 +3,14 @@
 
 package com.microsoft.azure.synapse.ml.causal
 
-import com.microsoft.azure.synapse.ml.core.contracts.{HasFeaturesCol, HasLabelCol, HasWeightCol}
+import com.microsoft.azure.synapse.ml.core.contracts.{HasFeaturesCol, HasWeightCol}
 import com.microsoft.azure.synapse.ml.param.EstimatorParam
-import org.apache.spark.ml.classification.{LogisticRegression, ProbabilisticClassifier}
-import org.apache.spark.ml.{Estimator, Model}
 import org.apache.spark.ml.ParamInjections.HasParallelismInjected
-import org.apache.spark.ml.param.shared.{HasMaxIter, HasPredictionCol}
-import org.apache.spark.ml.param.{DoubleArrayParam, DoubleParam, Param, Params}
+import org.apache.spark.ml.classification.{LogisticRegression, ProbabilisticClassifier}
+import org.apache.spark.ml.param.shared.HasMaxIter
+import org.apache.spark.ml.param.{DoubleArrayParam, DoubleParam, Params}
 import org.apache.spark.ml.regression.Regressor
-
-trait HasTreatmentCol extends Params {
```

Comment:
technically now that we are 1.0 this is a breaking change, if you could deprecate and create new ones would appreciate it

Reply:
This trait is shared between multiple estimators (DoubleML, SyntheticDiffInDiff, etc.), so I moved it to SharedParams.scala. The trait is still defined in the same package, so it shouldn't be breaking.